tools/build_config/Jenkinsfile

```diff
@@ -75,6 +75,22 @@ def get_default_herbert_branch(String job_name) {
   }
 }
 
+def get_herbert_ref_from_labels(String labels, String herbert_branch) {
+  /* Parse the string of Git issue labels for a label that matches Herbert_*.
+     If a match is found, build using the Herbert branch of that name. */
+  def match = (labels =~ "Herbert_([a-zA-Z0-9_-]+)")
+  try {
+    match[1]
+    return("Error: Found two Herbert branch labels on the pull request: " +
+           "'${match[0][1]}' and '${match[1][1]}'")
+  } catch (IndexOutOfBoundsException e1) {
+    try {
```

Comment:
a comment here noting that this is an expected exception because there should be only one label would be helpful; it's not often we want an exception to be thrown.